docker-compose.yml

```diff
@@ -32,8 +32,7 @@ controller:
     build: .
     dockerfile: Dockerfile.build
     entrypoint: /bin/bash
-    command: -c "sleep 5 && /go/src/github.com/shipyard/shipyard/controller/controller -D server --rethinkdb-addr rethinkdb:28015 -d tcp://swarm:2375"
-    working_dir: /go/src/github.com/shipyard/shipyard/controller
+    command: -c "make build && cd controller && ./controller -D server --rethinkdb-addr rethinkdb:28015 -d tcp://swarm:2375"
```

Comment:
We have to run `make build` from our default working directory `/go/src/github.com/shipyard/shipyard`.

We then have to run the built `controller` binary from `/go/src/github.com/shipyard/shipyard/controller` due to it serving the `static/` folder from a relative path, hence the addition of `cd controller`.